src/core/public/public.api.md

```diff
@@ -949,6 +951,15 @@ export interface ImageValidation {
     };
 }
 
+// @internal (undocumented)
+export interface InternalApplicationStart extends Omit<ApplicationStart, 'registerMountContext'> {
```

Comment:
@pgayvallet Is this okay to have it here? Had to do this to fix warnings with `scripts/check_published_api_changes.js --docs` script.

Reply:
The warning about the unexported `InternalApplicationStart` is preferable than exporting `InternalApplicationStart` from the index. `CoreSystem` is not really public anyway, so we care less about having exhaustive API doc for it than we do about not exporting internal contracts.

Reply:
Cool then, I will remove it from the exports. Thanks!